src/pom.js

```diff
@@ -182,6 +182,7 @@ foam.POM({
     { name: "foam/util/EmailTest",                                    flags: "js|java" },
     { name: "foam/util/PasswordTest",                                 flags: "js|java" },
     { name: "foam/util/SecurityUtilTest",                             flags: "js|java" },
+    { name: "foam/util/GeolocationSupport",                           flags: "js|java" },
```

Comment:
Could you move this to a sub-package like foam/util/geo or foam/geo and then give it its own POM (which you already have, just move it there)?